_includes/workshops/neurips/neurips2020.html

```diff
@@ -55,6 +57,12 @@ <h1 class="blog-title">Accepted Papers</h1>
                   <br>
                   <a target="_blank" href="../../../papers/{{paper.conference}}/{{paper.year}}/png/{{paper.poster}}"><img src="../../../assets/icons/research.png" height="30"></a>
                 </td>
+                <td style="max-width: 1px;" class="tg-0lax">
+                  <!--the style in the anchor allows to let the hyperlink continues in the next line, in other cases just breaks the table-->
+                  {{paper.publicationauthor}} ({{paper.year}}). <i>{{paper.title}}</i> [{{paper.type}} Presentation].
```

Comment:
We must include 'et al.' after the presenting author's name unless they were a single author on the paper to give appropriate credit to the others.